index.html

```diff
@@ -738,8 +738,8 @@ <h5>Fade both video and audio streams</h5>
       <dl>
           <dt>ffmpeg</dt><dd>starts the command</dd>
           <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
-          <dt>-filter:v "fade=in:st=0:d=1, fade=out:st=59:d=1"</dt><dd>applies a video filter that fades in the first and last 30 video frames</dd>
-          <dt>-filter:a "afade=in:st=0:d=1, afade=out:st=59:d=1"</dt><dd>applies an audio filter that fades in the first and last 30 video frames. <code>st</code> sets the start and <code>d</code> sets the duration.</dd>
+          <dt>-filter:v "fade=in:st=0:d=1, fade=out:st=59:d=1"</dt><dd>applies a video filter that fades in the first and last 30 video frames. <code>st</code> sets the start and <code>d</code> sets the duration. <code>st=59:d=1</code> is equivalent to one second</dd>
```

Comment:
The explanation here assumes that the input is one minute long and 30fps. So it would be good to state that the example is 1 minute long and 30fps, or preferably, remove the reference to 30fps altogether, and instead just refer to seconds. 'applies a video filter that fades in the first and last second. And instead of the last line, perhaps: '<code>st=59:d=1</code> this example uses a start time of 59 as it is a 60 second video'